main.go

```diff
@@ -81,7 +82,10 @@ func init() {
 	pflag.String("http-proto", "https", "Set to 'http' if necessary")
 	pflag.String("http-user", "", "HTTP username for pulling the remote file")
 	pflag.String("http-pass", "", "HTTP password for pulling the remote file")
+
 	pflag.String("http-url", "", "Remote endpoint to retrieve the file from")
+	pflag.String("s3-uri", "", "Remote endpoint in S3 to retrieve the file from if s3-arn isn't used")
```

Comment:
why not just arn? its a bit more verbose, but keeps it simpler

Reply:
As opposed to s3-arn or as opposed to also including s3-uri ?